website/identifiers/clients/datacite.py

```diff
@@ -46,9 +46,19 @@ def build_metadata(self, node):
             'publisher': 'Open Science Framework',
             'publicationYear': str(datetime.datetime.now().year),
             'resourceType': {
-                'resourceType': 'Project',
+                'resourceType': 'Registration',
```

Comment:
Should this be conditional based on `node.type`? Public projects can also have DOI's created, just on user input rather than automatically

Reply:
I haven't discussed this with product but I assume they'd agree.